### DISCUSSION TITLE
Flask-Login is ready for a 1.0 release

### DISCUSSION THREAD

**Discussion by alanhamlett** [maxcountryman/flask-login#670] [General]
There aren't any changes from v0.6.1 currently, but I think it's time for a 1.0 release. WDYT @maxcountryman @davidism?

**Comment by davidism** [comment]
Sounds good to me. I'd be interested in following up on what I was talking with @maxcountryman before about moving this to pallets-eco for long term community maintenance.

**Comment by tiesjan** [comment]
As a user of this library, I would like to suggest some removals prior to a 1.0 release:

- Since the _Login using Authorization header_ functionality is now deprecated, the functionality could be removed.
- Since the `LoginManager.setup_app` method is now deprecated, it could be removed.
- Unless there are serious plans to support authorization, the bullet point on potential _Flask-Principal_ support (at the top of the docs) could be removed.

I'd be happy to submit some patches if you agree, leading to a 1.0 release. Whether or not to move the project to @pallets-eco can be decided upon later.

**Reply by alanhamlett** [reply]
https://github.com/maxcountryman/flask-login/pull/675

**Comment by Zlopez** [comment]
Is there anything I can do to help to get the new release out of the door?
